packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -205,6 +206,8 @@ class TabController extends ChangeNotifier {
   /// [TabBarView.children]'s length.
   final int length;
 
+  final ObserverList<AnimationStartedListener> _animationStartedListeners = ObserverList<AnimationStartedListener>();
```

Comment:
Since now the animation maybe called with null duration which is not really an animation, maybe we should find a better name here....

Reply:
How about ```IndexChangeStartedListener```? I wonder if there's any naming standard in Flutter 🤔